docs/config.md

```diff
@@ -0,0 +1,182 @@
+# The go-ipfs config file
+The go-ipfs config file is a json document. It is read once at node instantiation,
+either for an offline command, or for starting the daemon. Commands that execute on
+a running daemon do not read the config file at runtime.
+
+## `Addresses`
+Contains information about various listener addresses to be used by this node.
+
+### `API`
+Multiaddr describing the address to serve the local http api on.
+
+Default: `/ip4/127.0.0.1/tcp/4001`
+
+### `Gateway`
+Multiaddr describing the address to serve the local gateway on.
```

Comment:
Link to Multiaddr. Multilink by using `[multiaddr][]` and then having `[multiaddr](//github.com/jbenet/multiaddr)` at the bottom.